pkg/azuredisk/nodeserver.go

```diff
@@ -187,6 +188,11 @@ func (d *Driver) NodePublishVolume(ctx context.Context, req *csi.NodePublishVolu
 		return nil, status.Error(codes.InvalidArgument, "Volume ID missing in request")
 	}
 
+	ephemeral := req.GetVolumeContext()["csi.storage.k8s.io/ephemeral"] == "true"
```

Comment:
where is this defined? will it change in the future? looks  like it would be broken easily.

Reply:
See the last sentence of the paragraph. 
https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/20190122-csi-inline-volumes.md#support-for-inline-csi-volumes

The parameter is handled in Kubernetes. https://github.com/kubernetes/api/blob/master/storage/v1beta1/types.go#L279-L304

Reply:
pls add `csi.storage.k8s.io/ephemeral` as a const and add links in the comments

Reply:
done